src/runtime_src/core/common/api/xrt_module.cpp

```diff
@@ -30,13 +31,70 @@
 # define AIE_COLUMN_PAGE_SIZE 8192
 #endif
 
-namespace {
+namespace
+{
 
 // Control code is padded to page size, where page size is
 // 0 if no padding is required.   The page size should be
 // embedded as ELF metadata in the future.
 static constexpr size_t column_page_size = AIE_COLUMN_PAGE_SIZE;
 
+// struct instr_buf - represent instruction buffer
+// TODO place holder in case we need special handling
+// TODO merge with ctrlcode?
+struct instr_buf
```

Comment:
struct instr_buf is identical to struct control_packet, why two structs?
also looks kind of similar to struct ctrlcode

Reply:
Keep only ctrlcode.